### PR TITLE
refactor: use createModal for creating tasks

### PR DESCRIPTION
**Pull request by MarinaPresmytskaia**
Rewrite taskModal (class component) on CreateModal (functionally component)
Add description input for board request
Uninstall sass

### CODE REVIEW

**Review by treyer — APPROVED** [review]

File: management-app/src/components/CreateModal/CreateModal.tsx [hunk 1]
```diff
@@ -84,7 +124,7 @@ function CreateModal({
           {titleModal}
         </Typography>
 
-        <Box component="form" onSubmit={onCreate} sx={{ Width: '600px' }}>
+        <Box component="form" onSubmit={handelSubmit} sx={{ Width: '600px' }}>
```
Comment (ElenaBezro) [inline management-app/src/components/CreateModal/CreateModal.tsx:127]: sx={{ Width: '600px' }} 
Width is ignored in this case because of the capital "W"

Comment (ElenaBezro) [reply management-app/src/components/CreateModal/CreateModal.tsx:127]: and for PIXELS you can omit quotation mark and 'px' like this:  sx={{ width: 600 }}

File: management-app/src/components/CreateModal/CreateModal.tsx [hunk 2]
```diff
@@ -29,11 +30,51 @@ function CreateModal({
...
+    (event: ChangeEvent<HTMLInputElement>) => {
+      const target = event.target as HTMLInputElement;
+      const value = target.value as string;
+
+      setTitleInput(value);
+    },
+    []
+  );
+
+  const handleChangeDescription = useCallback(
```
Comment (ElenaBezro) [inline management-app/src/components/CreateModal/CreateModal.tsx:53]: It seems it's no need to use useCallBack() in this case

File: management-app/src/components/CreateModal/CreateModal.tsx [hunk 3]
```diff
@@ -29,11 +30,51 @@ function CreateModal({
...
-  onChange,
+  isRenderDescription,
+  descriptionName,
+  labelDescription,
 }: TCreateModal) {
+  const [titleInput, setTitleInput] = useState<string>('');
+  const [taskDescription, setTaskDescription] = useState<string>('');
+  const [isDisabled, setDisabled] = useState<boolean>(true);
+
+  const handleInputChange = useCallback(
```
Comment (ElenaBezro) [inline management-app/src/components/CreateModal/CreateModal.tsx:43]: It seems it's no need to use useCallBack() in this case

File: management-app/src/components/CreateModal/CreateModal.tsx (hunk 1 above)
Comment (MarinaPresmytskaia) [reply management-app/src/components/CreateModal/CreateModal.tsx:127]: This code is  unnecessary, I deleted it. Thanks for our attention.

**Review by ElenaBezro — APPROVED** [review]

File: management-app/src/components/CreateModal/CreateModal.tsx (hunk 2 above)
Comment (MarinaPresmytskaia) [reply management-app/src/components/CreateModal/CreateModal.tsx:53]: Why?

File: management-app/src/components/CreateModal/CreateModal.tsx (hunk 3 above)
Comment (MarinaPresmytskaia) [reply management-app/src/components/CreateModal/CreateModal.tsx:43]: Why?

**Pull request merged**